README.md

```diff
@@ -619,6 +627,14 @@ Power available from time-varying generator (hydro, wind, solar) ![i](https://re
 at time ![t](https://render.githubusercontent.com/render/math?math=t).
 - ![x_{l}](https://render.githubusercontent.com/render/math?math=x_%7Bl%7D):
 Impedance of branch ![l](https://render.githubusercontent.com/render/math?math=l).
+- ![\underline{\delta}_{b, t}](https://render.githubusercontent.com/render/math?math=%5Cunderline%7B%5Cdelta%7D_%7Bb%2Ct%7D):
+Demand flexibility curtailments available at bus ![b](https://render.githubusercontent.com/render/math?math=b)
+at time ![t](https://render.githubusercontent.com/render/math?math=t).
+- ![\overline{\delta}_{b, t}](https://render.githubusercontent.com/render/math?math=%5Coverline%7B%5Cdelta%7D_%7Bb%2Ct%7D):
+Demand flexibility additions available at bus ![b](https://render.githubusercontent.com/render/math?math=b)
+at time ![t](https://render.githubusercontent.com/render/math?math=t).
+- ![\Delta](https://render.githubusercontent.com/render/math?math=%7B%5CDelta%7D):
+The length of the rolling load balance window, used to account for the duration that flexible demand is deviating from the base demand.
```

Comment:
Good points. I'm open to other variable name suggestions if there is something that you think fits particularly well. I initially settled on `Delta` because it aligned with the similar `deltas` I was using for demand shifting. As for this parameters becoming a variable, I certainly think that's something we can explore in the future. If it's alright with you, I think we should keep it as a parameter for now, especially while we're trying to integrate other flexibility profiles and aggregate those profiles by sector.

Reply:
Other `deltas` are `MW`, this `Delta` is hours?

Reply:
Correct. I'll be sure to specify those units. "Aligned" probably wasn't the right word above. I wasn't trying to say that these variables/parameters are similar, rather that they pertain to a similar part of the model (demand flexibility).

Reply:
I agree with Bainan, it would be nice to be more descriptive here. Maybe `Delta_t^{balance}` or something similar?

Reply:
Done.